### PR TITLE
chore(AXO): Remove Accelerated XO

### PR DESCRIPTION
**Pull request by mnicpt**
### Description
Deprecation of Accelerated Checkout V1.
https://paypal.atlassian.net/browse/DTXOUP-985

### Testing
1. Run SPB with current version of `paypal-checkout-components`.
2. Run SPB with current version of `checkoutuinodeweb`.

**Results**
No errors with member checkout, inline guest checkout rendering.

### Groups who should review (if applicable)

❤️  Thank you!
#585

### CODE REVIEW

**Review by wsbrunson — APPROVED** [review]

File: test/client/actions.js [hunk 1]
```diff
@@ -26,172 +25,6 @@ import {
...
-
-                if (!orderID) {
-                    throw new Error(`Expected orderID to be returned by actions.order.create`);
-                }
-
-                return orderID;
-            }));
-
-            window.xprops.onApprove = mockAsyncProp(expect('onApprove', async (data) => {
-                if (!data.accelerated) {
```
Comment (elizabethmv) [inline test/client/actions.js:66]: Does the whole test need to be removed or just the accelerated check? It doesn't seem like the whole test is specific to AXO

**Review by elizabethmv — APPROVED** [review]
Lgtm! One question about a test

**Comment by gregjopa** [comment]
Hi @mnicpt, thanks for cleaning up this code!

I did notice a couple more spots. Can you send a PR to remove inline from these two files?
-  https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/api/order.js#L997-L1006
- https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/button/pay.js#L84

**Comment by mnicpt** [comment]
> Hi @mnicpt, thanks for cleaning up this code!
> 
> I did notice a couple more spots. Can you send a PR to remove inline from these two files?
> 
> * https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/api/order.js#L997-L1006
> * https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/button/pay.js#L84

@gregjopa Those are not `inlinexo` but rather for the `inline` flow property. 
image

File: test/client/actions.js (hunk 1 above)
Comment (mnicpt) [reply test/client/actions.js:66]: @elizabethmv The whole test. I created both of those specific for AXO.

**Pull request merged**